### DISCUSSION TITLE
E-arsiv ve E-fatura farkı

### DISCUSSION THREAD

**Discussion by edukah** [mlevent/fatura#50] [General]
Bu konu için doğru yer burası mı bilmiyorum ama yazmak istedim. Muhasebe konusuna çok uzağım google'dan araştırdığım kadarıyla E-arsiv ve E-fatura biraz farklı. E-fatura GİB sistemine kayıtlı kişilere kesilirken E-arsiv kayıtlı olmayan kişilere oluşturuluyor. Fatura oluşturmadan önce kişinin GIB'e kaydını kontrol ettikten sonra devam etmek gerekmiyor mu? Uygulamada oluşturduğumuz e-fatura mı oluyor, e-arsiv oluşturamıyor muyuz?

**Comment by mlevent** [comment]
merhabalar,

e-fatura mükellefiyseniz bu paketi kullanamazsınız. fakat e-arşiv fatura, serbest meslek makbuzu, müstahsil makbuzu kesmek için kullanabilirsiniz.

bu durumda, eğer e-arşiv kullanıcısıysanız, karşı taraf ister e-fatura mükellefi olsun, ister e-arşiv, ister x şahıs; herhangi bir kontrol yapmanıza gerek kalmadan e-arşiv fatura kesebilirsiniz.

kısacası, halihazırda faturalarınızı gib e-arşiv portal üzerinden kesiyorsanız, paketi kullanabilirsiniz.
